Dockerfile

```diff
@@ -1,16 +1,17 @@
 FROM ubuntu:16.04
-MAINTAINER Haris Amin <aminharis7@gmail.com>
+LABEL maintainer="Haris Amin <aminharis7@gmail.com>"
+LABEL Description="Docker Container for the Apple's Swift programming language"
 
 # Install related packages and set LLVM 3.6 as the compiler
 RUN apt-get -q update && \
+    apt-get dist-upgrade -y && \
```

Comment:
Please do not use `dist-upgrade` in submissions to the official images.

Reply:
See https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#apt-get, especially:

> You should avoid RUN apt-get upgrade or dist-upgrade, as many of the “essential” packages from the parent images won’t upgrade inside an unprivileged container. If a package contained in the parent image is out-of-date, you should contact its maintainers. If you know there’s a particular package, foo, that needs to be updated, use apt-get install -y foo to update automatically.

Reply:
@hamin I've updated the pull request to be compliant with docker official image best practices.

Reply:
Much better :+1: